main/src/util/utils.hpp

```diff
@@ -2,6 +2,7 @@
 
 #include <tuple>
 #include <omp.h>
+#include <version.h>
```

Comment:
use quotes here, "version.h". The <> syntax is used for system and external library includes.